DQM/L1TMonitorClient/src/L1TOccupancyClient.cc

```diff
@@ -543,7 +541,8 @@ double L1TOccupancyClient::xySymmetry(const ParameterSet& ps,
     if (verbose_) {
       cout << "statsup= " << statsup << ", statslow= " << statslow << endl;
     }
-    enoughStats = TMath::MinElement(nActualStrips, maxAvgs.get()) > TMath::Max(statsup, statslow);
+    if (nActualStrips > 0)
+      enoughStats = TMath::MinElement(nActualStrips, maxAvgs.get()) > TMath::Max(statsup, statslow);
     if (verbose_) {
       cout << "stats: " << TMath::MinElement(nActualStrips, maxAvgs.get())
```

Comment:
same here, call to `TMath::MinElement` should be avoid for `nActualStrips==0`

Reply:
As above